### ISSUE TITLE
TalkBack should have a context menu for selection if supported

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#304]
```
What steps will reproduce the problem?
1. Open the local context menu while focused on a view with an 
AccessibilityNodeInfo that supports ACTION_SET_SELECTION (and possibly the 
granularity actions too).

What is the expected output?
A context menu with Start/Stop Selection options (amongst other things).

What do you see instead?
You don't see these options unless the widget you're focused on is an 
android.widget.EditText.


Without this, you can't really support selection on some arbitrary widget with 
text.


```

Original issue reported on code.google.com by `ma...@maxli.ca` on 8 Aug 2013 at 1:11

**Comment by GoogleCodeExporter** [comment]
```
This is supported in TalkBack 3.5.0 under the "Cursor control" menu item on 
Android 4.3+.
```

Original comment by `caseybur...@google.com` on 5 Dec 2013 at 9:17
- Changed state: **Done**
